helix-view/src/document.rs

```diff
@@ -664,6 +668,11 @@ impl Document {
         }
     }
 
+    /// Mark document as recent used for MRU sorting
+    pub fn mark_as_used(&mut self) {
```

Comment:
Should this also be called in `Editor::focus` and `Editor::focus_next` so that the focused-at time is updated when switching between splits/windows?

Reply:
I'm don't know how it must in common case. In my personal usage of vim each "focus on buffer" will mark that buffer as "used". 

Added set focused-at time on `Editor::focus` and `Editor::focus_next`